### DISCUSSION TITLE
2006/08/08/labview-%E4%B8%AD%E7%9A%84%E6%95%B0%E5%AD%97%E5%9E%8B%E6%95%B0%E6%8D%AE-2-%E8%BF%90%E7%AE%97

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#104] [Announcements]
# 2006/08/08/labview-%E4%B8%AD%E7%9A%84%E6%95%B0%E5%AD%97%E5%9E%8B%E6%95%B0%E6%8D%AE-2-%E8%BF%90%E7%AE%97

1.  常用函数

https://blog.qizhen.xyz/2006/08/08/labview-%E4%B8%AD%E7%9A%84%E6%95%B0%E5%AD%97%E5%9E%8B%E6%95%B0%E6%8D%AE-2-%E8%BF%90%E7%AE%97

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2008年06月11日 08:44 编辑
非常感谢指正，我把改正了 :)

回复
jing
2008年06月11日 02:17 编辑
指出一个可能是小小的笔误，在第2部分的最后一段中“在公式节点中只允许有一个字符串，代表输入参数，例如本例中，参数用 f 表示。LabVIEW 在线帮助里列出有表达式节点所支持的运算符、函数和表达式规则。”这里应该是“表达式节点”，而不是“公式节点”。

回复
jing
2008年06月11日 01:51 编辑
指出一个小问题：在第2部分的最后一段中“ 在公式节点中只允许有一个字符串，代表输入参数，例如本例中，参数用 f 表示。LabVIEW 在线帮助里列出有表达式节点所支持的运算符、函数和表达式规则。”这里是否应该是表达式节点，而不是公式节点呢？呵呵，仔细看应该能看出来，不过还是指出一下，方便后来人。

回复
Qizhen
2006年08月9日 12:59 编辑
工作之后和当学生的时候心态就不一样了，学生需要多一些证书作为找工作的资本。
工作以后，就算再应聘，别人主要看的是你以往的工作经验而不是证书，所以就不太关心这些认证了。

回复
Wenxing
2006年08月9日 12:10 编辑
是NI的程序员认证，或者是中级认证，我感觉你已经很厉害了，为什么没有去考证呢？

回复
Qizhen
2006年08月9日 09:28 编辑
不好意思啊，我不知道 CLD 认证是什么。

回复
Wenxing
2006年08月8日 18:02 编辑
第一次来你的space，感觉内容非常丰富。
我有个问题，要通过CLD认证需要多少时间的实践，对于一个再读学生而言是不是不太现实呢？

回复
